packages/neos-ui-ckeditor5-bindings/src/EditorToolbar/Helpers/renderToolbarComponents.js

```diff
@@ -41,6 +41,11 @@ export default richtextToolbarRegistry => {
                     [callbackPropName]: e => e.stopPropagation() || executeCommand(commandName, ...commandArgs)
                 };
 
+                // filter out props that are not needed by the component and lead to a render warning
+                if (commandName === 'code') {
```

Comment:
why does this have to be conditionally wrapped? And what does `code` mean?

Reply:
The component that is used for the code command (class U) is not using the two props, and they end up in the DOM and that leads to the issue. In other places, we also omit properties to prevent that they end up in the IconButtons of the toolbar.

Reply:
Does it mean the "code" Dom element specifically or some CK internal thing?
If it's a CK internal, a reference in the comment would be great. Like this it just looks like a random magic string.

Reply:
The formatting type code comes from CKEditor and we just pass thru the props to the components.

Reply:
We actually use just code. But the newer CKEditor also has `codeBlock`, but we just don`t have it yet.